src/main.ts

```diff
@@ -1,17 +1,31 @@
 import { createApp } from 'vue'
-import { createHead } from "@vueuse/head"
+import { createHead } from '@vueuse/head'
+import { createI18n } from 'vue-i18n'
 
 import App from './App.vue'
 import router from './router'
-import ComponentLoader from "@/components/loader";
+import ComponentLoader from '@/components/loader'
 
+import { en, pt_BR } from '../src/locales'
 // import './assets/css/index.css'
 
+const languages = {
+    en,
+    pt_BR,
+}
+
+const i18n = createI18n({
+    locale: 'en',
+    fallbackLocale: 'en',
+    messages: languages,
+})
+
 const app = createApp(App)
 const head = createHead()
 
 app.use(router)
 app.use(head)
```

Comment:
Can you better explain the usage of `unhead`?

Reply:
The unhead was not actually used by me, it was already being called when I made the clone of the repository and ended up leaving it